src/store/index.ts

```diff
@@ -32,7 +32,7 @@ import safe, { SAFE_REDUCER_ID, SafeReducerMap } from 'src/routes/safe/store/red
 import transactions, { TRANSACTIONS_REDUCER_ID } from 'src/routes/safe/store/reducer/transactions'
 import { NFTAssets, NFTTokens } from 'src/logic/collectibles/sources/OpenSea'
 
-export const history = createHashHistory({ hashType: 'slash' })
+export const history = createHashHistory()
```

Comment:
so 'slash' was the default value. Good to know.

Reply:
I updated history to 5.0.0 and it didn't have such prop, this is why removed it. But true, I checked and slash is the default value. Definitely good to know.